cmd/mungedocs/mungedocs.go

```diff
@@ -28,6 +29,8 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+const latestReleaseBranch = "release-1.1"
```

Comment:
@ihmccreery, I want add a note saying this variable needs to be updated when we officially publish a new release series (vX.Y), which doc shall I add it to? https://github.com/kubernetes/kubernetes/blob/master/docs/devel/releasing.md?